src/main/java/com/google/flourbot/execution/MacroExecutionModuleImplementation.java

```diff
@@ -159,4 +170,9 @@ private String getDate(String pattern) {
     DateTimeFormatter myFormatObj = DateTimeFormatter.ofPattern(pattern);
     return myDateObj.format(myFormatObj);
   }
+
+  private String selectRandomFromData(List<String> data) {
```

Comment:
Don't add private methods if they are not used yet, we can do it in the next change.